src/schema/ecommerce/query_helpers.ts

```diff
@@ -48,3 +48,96 @@ export const BuyerSellerFields = gql`
     }
   }
 `
+
+export const BuyerOrderFields = gql`
+  ${BuyerSellerFields}
+  ${RequestedFulfillmentFragment}
+  buyerPhoneNumber
+  buyerTotalCents
+  code
+  commissionFeeCents
+  commissionRate
+  createdAt
+  currencyCode
+  displayCommissionRate
+  id
+  itemsTotalCents
+  lastApprovedAt
+  lastSubmittedAt
+  mode
+  sellerTotalCents
+  shippingTotalCents
+  state
+  stateExpiresAt
+  stateReason
+  stateUpdatedAt
+  taxTotalCents
+  transactionFeeCents
+  updatedAt
+  lineItems {
+    edges {
+      node {
+        id
+        priceCents
+        artworkId
+        editionSetId
+        quantity
+      }
+    }
+  }
+`
+
+export const SellerOrderFields = gql`
+  ${BuyerSellerFields}
```

Comment:
#minor but  given the pattern (`BuyerOrderFields`, `SellerOrderFields`), `BuyerSellerFields` is a little confusing. Maybe we can call it `ParticipantFields`?